src/shared/directives/substitutor/label-substitutor.directive.spec.ts

```diff
@@ -18,9 +18,10 @@ import createSpyObj = jasmine.createSpyObj;
 })
 class TestHostComponent {
 
-    @Input() caseField: CaseField;
-    @Input() eventFields: CaseField[];
-    @Input() formGroup: FormGroup;
+  @Input() caseField: CaseField;
+  @Input() eventFields: CaseField[];
+  @Input() formGroup: FormGroup = new FormGroup({});
+  @Input() isEmptyIfPlaceholderMissing: Boolean = false;
```

Comment:
Is this ever set to true? I cannot see it being set on any template and within the component/service code.